eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -38,12 +38,13 @@ param(
   [Parameter(Mandatory = $true)]
   [string] $OutputLocation
 )
-Set-StrictMode -Version 3
 . $PSScriptRoot/common.ps1
 . $PSScriptRoot/Helpers/PSModule-Helpers.ps1
```

Comment:
We should eventually fix the strict mode issues for this script instead of working around it.